specs/latest/2.0/index.html

```diff
@@ -3386,6 +3492,19 @@ <h2 class="no-toc">Table of contents</h2>
         but that can be achieved by adjusting the data store offset accordingly.
     </div>
 
+    <h3><a name="DOM_UPLOAD_UNPACK_PARAMS">Pixel store parameters for <code>TexImageSource</code> uploads</a></h3>
+    <p>
+        <code>UNPACK_ALIGNMENT</code> and <code>UNPACK_ROW_LENGTH</code> are
```

Comment:
We need to clarify the situation with TexSubImage from TexImageSource.  Originally we took height/width from the TexImageSource, but now we try to use a sub region of the source, we need a way to define this sub region.  If SKIP_PIXELS and SKIP_ROWS are used to define xoffset and yoffset, how do we specify the subSource's width and height? If they become the function args, then I suggest we also take xoffset and yoffset args and just ignore SKIP_PIXELS and SKIP_ROWS as well.  It's bad idea to split specifying an sub-source using both function args and unpack params.